raw_data/preprocessed/africa/guinea-bissau_portuguese.txt

```diff
@@ -0,0 +1,199 @@
+Elementos da Ofensa Ecológica
+(26) Constituem, designadamente, os elementos das ofensas ecológicas: [a] poluição atmosférica; [b] poluição hídrica; [c] degradação do solo e de sub-solo; [d] degradação da flora e da fauna; [e] poluição química; [f] ofensa da paisagem; [g] poluição sonora; [h] perturbação dos níveis de luminosidade; [i] poluição industrial.
+
+Proibição de importação de resíduos ou lixos perigosos
+(27) É expressamente proibida a importação de resíduos ou lixos perigosos no espaço sob a jurisdição nacional.
+
+CAPÍTULO III - DOS INSTRUMENTOS DA POLÍTICA E DA GESTÃO AMBIENTAL
+
+Os Instrumentos
+(28) São instrumentos da política e da gestão ambiental, designadamente os seguintes: [a] Plano Nacional de Gestão Ambiental; [b] Zonas de Protecção Especial; [c] Detecção Remota e Sistema de Informação Geográfica (DT&SIG); [d] Licenciamento Ambiental; [e] Avaliação Ambiental; [f] Auditoria Ambiental; [g] Políticas e Planos de Ordenamento do Território, incluindo a classificação e criação de áreas, reservas ou paisagens protegidas com regimes especiais de conservação; [h] A Estratégia Nacional de Conservação da Natureza integrada na estratégia Africana e Mundial; [i] Informação, Educação e Comunicação Ambiental; [j] O Sistema Nacional de Protecção Civil e Prevenção dos Incêndios; [k] Convenções, Tratados e Acordos Internacionais; [l] Informações sobre os Dados Meteorológicos e Climáticos. [2] Lei especial definirá as áreas e zonas de maior “impacte” ambiental, onde se fará controlo e se tomarão medidas permanentes que normalizem a qualidade do ambiente.
+
+Fundo Ambiental
+(29) [1] É criado um Fundo Ambiental, cuja gestão será objecto de um diploma próprio. [2] Integra designadamente o Fundo Ambiental as dotações do Orçamento Geral do Estado, as compensações, doações e valores provenientes de aplicação das multas e em consequência da violação das normas ambientais.
+
+Conservação da Natureza
+(30) Para enquadramento e utilização das políticas globais do ambiente com as sectoriais será elaborada pelo organismo responsável pela área do ambiente, no prazo de um ano após a publicação do presente diploma, a estratégia nacional de conservação da natureza, que será submetida à autoridade competente para aprovação.
+
+Áreas Protegidas, Reservas, Sítios, Conjuntos e Objectos Classificados
+(31) Legislação especial definirá a implementação e regulamentação de uma rede nacional contínua de áreas protegidas.
+
+Avaliação Ambiental
+(32) [1] Os planos, projectos, programas, trabalhos e acções que possam afectar o ambiente e a qualidade de vida das pessoas, quer sejam da responsabilidade e iniciativa de um organismo da administração central, regional ou local, quer de instituições públicas ou privadas, devem respeitar as preocupações e normas desta lei, bem como da lei de avaliação ambiental e terão de ser acompanhados de um estudo de ''impacte'' ambiental. [2] A avaliação ambiental e o conteúdo do seu procedimento será regulado pela legislação própria.
```

Comment:
The quotations around ''impacte'' are different from the ordinary quotations (") used.